dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHV2IT.java

```diff
@@ -1,15 +1,12 @@
 package io.dockstore.client.cli;
 
-import java.io.BufferedReader;
-import java.io.InputStreamReader;
```

Comment:
Did we not have a rule that prevents unused imports?

Reply:
It's probably not on for tests, we can give it a shot